aea/connections/stub/connection.yaml

```diff
@@ -2,7 +2,7 @@ name: stub
 author: fetchai
 version: 0.1.0
 license: Apache-2.0
-fingerprint: ""
+fingerprint: {'__init__.py': 'QmWwepN9Fy9gHAp39vUGFSLdnB9JZjdyE3STnbowSUhJkC', 'connection.py': 'QmYBKp9zhAmv21u7JJ7XsTgQFtjfY2FMtfhBDJTCJxcSVW'}
```

Comment:
All finished packages now have the fingerprints of their files as part of the config. This will be helpful for ensuring their integrity.

Reply:
This is generated automatically right ? (Just a sanity check :D )